src/main/java/utils/InputValidation.java

```diff
@@ -0,0 +1,93 @@
+package utils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class InputValidation {
+
+    private static final int LOTTO_MINIMUN_NUMBER = 1;
+    private static final int LOTTO_MAXIMUN_NUMBER = 45;
+    private static final int LOTTO_SIZE = 6;
+    private static final int LOTTO_PRICE = 1000;
+    private static final String LOTTO_NUMBER_DELIMITER = ", ";
+    private static final String ERROR_PRICE_NON_INTEGER = "가격은 정수만 가능합니다.";
+    private static final String ERROR_NEGATIVE_INTEGER = "가격은 1000원 이상만 가능합니다.";
+    private static final String ERROR_NUM_OF_BALL = "로또 번호는 6개의 번호를 입력해줘야 합니다.";
+    private static final String ERROR_BALL_NON_INTEGER = "로또 번호는 숫자만 입력해줘야 합니다.";
+    private static final String ERROR_NUMBER_OUT_RANGE = "로또 번호는 1에서 45 사이의 값을 입력해줘야 합니다.";
+    private static final String ERROR_DUPLICATE_NUMBER = "로또 번호는 중복되면 안 됩니다.";
+
+    public static int validatePrice(final String inputPrice) {
+        final int price = checkNonInteger(inputPrice, ERROR_PRICE_NON_INTEGER);
+        checkUnderMinimumPrice(price);
+
+        return price;
+    }
+
+    public static List<Integer> validateWinningNumber(final String inputNumbers) {
+        final List<String> splitedNumbers = Arrays.asList(inputNumbers.split(LOTTO_NUMBER_DELIMITER));
```

Comment:
```suggestion
        final List<String> splitNumbers = Arrays.asList(inputNumbers.split(LOTTO_NUMBER_DELIMITER));
```

<img width="1041" alt="스크린샷 2022-02-24 오후 10 40 06" src="https://user-images.githubusercontent.com/18049492/155535049-ebe2870b-f5b0-45e8-9077-a7d1fb07fa49.png">

Reply:
해당 메서드명의 문법적 오류도 올바르게 수정했습니다!!